extensions/wikia/PolldaddyTag/PolldaddyTagValidator.class.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+class PolldaddyTagValidator {
+
+	public static function validateAttributes( $params ) {
+		return array_key_exists( 'id', $params ) && !empty( $params['id'] );
```

Comment:
I could agree that outside the `empty()` function, checking `$params['id']` while `id` key doesn't exist, would throw `E_NOTICE : type 8 -- Undefined index: id -- at line 6`. However, it is safe to do such check using `empty()`. You can drop ~~the base~~ `array_key_exists` check.